docs/Activity3_Answers.md

```diff
@@ -0,0 +1,71 @@
+# Activity 3
+
+### 1. How do you ensure that users that navigate to the protected pages cannot bypass authentication requirements?
```

Comment:
Please also note that we require the token to be included with all requests on the backend (I will have a PR up with this change shortly). React runs on the frontend, so we cannot trust security validation on it alone.

Reply:
Based on the feature/backend-upload-route  PR, I have updated the answer regarding this comment. 

(commit 4bc5276f6388e73d342f9658d63a8cf30c397a74)